packages/utils/scaffold.js

```diff
@@ -50,7 +50,7 @@ module.exports = function runTransform(webpackProperties, action) {
 			transformations.push("merge");
 		}
 		const ast = j(
-			initActionNotDefined
+			addActionDefined
```

Comment:
Why are you changing this btw?

Reply:
Because in cases other than add (init and remove), we're starting from "module.exports = {}" and not reading existing config. We're regenerating the whole config file by using `addProperty`. The generator would directly clear the things to remove and the newly generated config file would be created based on this it.